package.json

```diff
@@ -23,6 +23,9 @@
   "pre-commit": "check",
   "homepage": "https://github.com/Real-Dev-Squad/website-dashboard#readme",
   "devDependencies": {
+    "@babel/core": "^7.26.0",
+    "@babel/preset-env": "^7.26.0",
+    "babel-jest": "^29.7.0",
```

Comment:
I don't see we are using these packages in code anywhere, then are we installing it

If not needed please remove it